src/controller/java/src/chip/devicecontroller/model/ClusterState.java

```diff
@@ -51,6 +57,35 @@ public Optional<Long> getDataVersion() {
     return dataVersion;
   }
 
+  /**
+   * Convenience utility for getting all attributes in Json string format.
+   *
+   * @return all attributes in Json string format., or empty string if not found.
+   */
+  public String getAttributesJson() {
+    JSONObject combinedObject = new JSONObject();
+    Stream<JSONObject> attributeJsons =
+        attributes.values().stream().map(it -> it.getJson()).filter(it -> it != null);
+
+    attributeJsons.forEach(
+        attributes -> {
+          for (Iterator<String> iterator = attributes.keys(); iterator.hasNext(); ) {
+            String key = iterator.next();
+            if (combinedObject.has(key)) {
+              Log.e(TAG, "Conflicting attribute tag Id is found: " + key);
+              return;
```

Comment:
use continue because of for loop, not retur.

Reply:
done, thanks